Index: LibreELEC-settings-37e206b712a46213522a567a408382f5f082502a/Makefile
===================================================================
--- LibreELEC-settings-37e206b712a46213522a567a408382f5f082502a.orig/Makefile
+++ LibreELEC-settings-37e206b712a46213522a567a408382f5f082502a/Makefile
@@ -4,7 +4,7 @@
 
 ADDON_NAME=service.libreelec.settings
 ADDON_VERSION=9.8.0
-DISTRONAME:=LibreELEC
+override DISTRONAME=LibreELEC
 
 SHELL=/bin/bash
 BUILDDIR=build
